website/content/en/docs/getting-started/getting-started-with-karpenter/_index.md

```diff
@@ -11,7 +11,7 @@ Karpenter automatically provisions new nodes in response to unschedulable pods.
 
 This guide shows how to get started with Karpenter by creating a Kubernetes cluster and installing Karpenter.
 To use Karpenter, you must be running a supported Kubernetes cluster on a supported cloud provider.
-Currently, only EKS on AWS is supported.
+Currently, not only is EKS on AWS supported, but also Azure.
```

Comment:
```suggestion
Currently, the following Cloud Providers are supported:
- [AWS](https://github.com/aws/karpenter-provider-aws)
- [Azure](https://github.com/Azure/karpenter-provider-azure)
```

I'd prefer that we just list down the cloudproviders that we do support if we can